docs/deployment.md

```diff
@@ -0,0 +1,52 @@
+# Deployment
+
+## Requirements
+
+- K8s cluster
+- Helm
+- Ingress controller (e.g. Traefik)
+- Nvidia GPU Operator
+
+## Artifacts
+Prem-Operator has three artifacts:
+1. [Source Code](https://github.com/premAI-io/saas-controller)
+1. [Prem-Operator Docker image](#container-images)
+1. [Prem-Operator Helm chart](https://hub.docker.com/r/premai/prem-operator-chart)
+
+## Installation
+After setting up K8s cluster, install [Nvidia GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html) and [Traefik](https://doc.traefik.io/traefik/getting-started/install-traefik/#use-the-helm-chart) as an ingress controller.
+Note that Nvidia GPU Operator is required for GPU support and Traefik is required for handling ingress traffic.
```

Comment:
These are recommendations, but it is good to point out this is what we have tested it with

Reply:
For instance add "optionally", before install and change "Traefik is required" to "Traefik is on option"

Reply:
Done